.github/workflows/typedoc.yml

```diff
@@ -1,13 +1,17 @@
 name: Typedoc
 
 on:
-  push:
+  pull_request:
+    types: [closed]
     branches:
-      - master  # Set a branch name to trigger deployment
-      - typedoc-action
+      - dev
+  push:
+     branches:
+       - typedoc-action
```

Comment:
This was the branch where this action was originally added, so this can be removed. I think any push/PR merge to dev should trigger this action.

Reply:
Should we update typedocs directly on any push to dev, or could that have other cascading effects as suggested by @tnorling ?